tests/integration/nm/ovs_test.py

```diff
@@ -378,6 +378,102 @@ def ovs_bridge_over_bond_system_iface_with_same_name(eth1_up, eth2_up):
     time.sleep(1)
 
 
+@pytest.fixture
+def ovs_bridge_first_and_ovs_interface_with_same_name_ipv4():
+    # The order on this function is important. The OVS bridge must be defined
+    # before the OVS interface.
+    cmdlib.exec_cmd(
+        "nmcli connection add type ovs-port conn.interface br-ex master br-ex "
+        "con-name ovs-port-br-ex".split(),
+        check=True,
+    )
+    cmdlib.exec_cmd(
+        "nmcli connection add type ovs-bridge con-name br-ex conn.interface "
+        "br-ex".split(),
+        check=True,
+    )
+    nmcli_ovs_interface = (
+        "nmcli",
```

Comment:
Does black require all items individually? If not, you can make this more readable as

```
nmcli connection add
type ovs-interface
slave-type ovs-port
conn.interface br-ex
...
```

Reply:
Nope, that is not accepted by black. Black requires all items individually